src/eterna/util/ExternalInterface.ts

```diff
@@ -2,16 +2,21 @@ import * as log from 'loglevel';
 import {Registration, UnitSignal} from 'signals';
 import {Deferred, Assert} from 'flashbang';
 
+// We have to deal with callbacks weakly. Ideally there'd be a mechanism to make this a bit
+// more explicit using `unknown`, but I couldn't immediately figure out a way to do it
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type AnyFunction = (...args: any[]) => any;
```

Comment:
yeah for now I think it's reasonable to chuck your `any` violations into one place for safekeeping